### PR TITLE
To do list for review

### PR DESCRIPTION
**Pull request by alex1779**
In this project, I will build a simple HTML list of To Do tasks. The list will be styled according to the specifications listed later in this lesson. This simple web page will be built using webpack and served by a webpack dev server.

Adding index html, css style sheet, js file, webpack configuration, dist files.
Adding license file.
Update readme file.
Adding linters files.

Added the function to add element to the list.
Remove function to the list.
Editing item from the list.
Remove function to the list.
Update items object's value for completed key upon user actions.

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -0,0 +1,200 @@
+class Task {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+
+function restoreOriginalIcons() {
```
Comment (Estete9) [inline src/utils.js]: Great work Alejandro! I like the solutions you applied; the ES6 syntax is very well done.
What do you think of refactoring your functions into arrow functions? That way you can keep with that ES6 syntax
ex: 
`const restoreOriginalIcons = () => {//your code}`

**Review by Estete9 — COMMENTED** [review]
Great job Alejandro 🥳 
Almost perfect! 💯 
I really like that you have only the necessary in index.js and that all your ES6 syntax is correctly used.

I would recommend looking into refactoring your functions into arrow functions so the syntax stays congruent with ES6

File: src/utils.js (hunk 1 above)
Comment (alex1779) [reply src/utils.js]: It is a good recommendation, Thanks!

File: src/utils.js [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+};
+
+function setValueTextArea(textArea, labelTask) {
+  labelTask.innerHTML = textArea.value;
+  textArea.style.display = 'none';
+  labelTask.style.display = 'block';
+  restoreOriginalIcons();
+  restoreBackgroundWhite();
+  scrapeAndSave();
+}
```
Comment (Estete9) [inline src/utils.js:44]: Great job with refactoring your code 🥳 
OPTIONAL:
I would suggest that you make all your functions into arrow function.
In this case specifically, you can use the sintax: `const setValueTextArea = (textArea, labelTask) => { YOUR CODE HERE }`

**Review by Estete9 — APPROVED** [review]
Great job my friend! 🍾 

You successfully corrected and applied ES6 syntax and the DRY, KISS, YAGNI principles.

You have some optional improvements but your PR is approved!

**Pull request merged**